docs/user-guide/config-examples.md

```diff
@@ -32,12 +33,13 @@ all of the nixvim options are accessible no matter how you are using it (flake,
 | [Tanish2002/neovim-config] | |
 | [traxys/Nixfiles] | |
 | [veeronniecaw/ronvim] | |
-| [zainkergaye/nixosdotfiles] | NixVim editor primarily for Java and Asciidoc | 
+| [zainkergaye/nixosdotfiles] | NixVim editor primarily for Java and Asciidoc |
```

Comment:
Although trailing whitespace isn't ideal, I'd rather avoid fixing it in this commit as it means this commit is no longer single-purpose.

I'm fine for it do be done in this PR as a separate commit though :slightly_smiling_face:

Reply:
Sounds good, I will separate this out. My config removes whitespaces on save so it shipped with the same commit!